config/api.yml

```diff
@@ -2443,17 +2443,17 @@
     :collection_actions:
       :get:
       - :name: read
-        :identifier: control_explorer_view
+        :identifier: miq_action_show_list
```

Comment:
For these next three, what was the rational for using this permission?

The reason I ask is that this was the other permission that I had a question on, since in https://github.com/ManageIQ/manageiq/pull/20438 it was removed:

https://github.com/ManageIQ/manageiq/pull/20438/files#r525547913

Your response really didn't provide much context.  Am I to assume that `miq_action_show_list` is the same permission then?

Reply:
Anything containing `explorer` in the API feature identifiers is probably wrong. The explorer is a (bad) UI concept and these features should not be tied to the API at all. The endpoint here is retrieving you a list of policy actions, for this we have this dedicated RBAC feature identifier...

Reply:
"Probably wrong" or not, there are probably deployments of MIQ where this is currently the assumed as the identifier gating these permissions.

If this changes, this then either enables or disables configured permissions for some users based on what is currently configured, making it a breaking change.

This is similar to what I am getting at here:  https://github.com/ManageIQ/manageiq-api/pull/966#discussion_r526986231

Reply:
Yeah, this will also need a migration as it was missed in the original PRs :confused:

Reply:
@NickLaMuro @skateman prior to Control explorer split and renaming of the product features we only had a parent feature exposed in product features tree and that was `control_explorer` any children features under that used to be hidden features and were granted access if user role had access to `control_explorer`. User roles only had `control_explorer` feature that is renamed to `control`